scripts/test_grok1_multiblock_experiment.py

```
@@ -1296,6 +1413,318 @@ def test_int4_evidence_only_requires_hp_123(self) -> None:
+        issue, agent = multiblock._agent_for_args(args)
+        self.assertIn("#85", issue)
+        self.assertIn("RM-608", issue)
+        self.assertIn("#85", agent)
+
+    def test_same_budget_int4_baseline_cites_issue_85(self) -> None:
+        """The 8192 evidence-only INT4 control is #85 evidence, not a #80 run."""
+        baseline = argparse.Namespace(
+            arm="int4", tokens=8192, evidence_only=True, hp_blocks=None
+        )
+        self.assertTrue(multiblock._is_v4_run(baseline))
+        issue, agent = multiblock._agent_for_args(baseline)
```

Ruff (0.16.1)

[warning] Unpacked variable `agent` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)